cpp/open3d/t/io/file_format/FilePLY.cpp

```diff
@@ -210,30 +237,38 @@ bool ReadPointCloudFromPLY(const std::string &filename,
                         "size of {} ({}).",
                         name, size, element_name, element_size);
             }
+
             const std::string attr_name = std::string(name);
 
-            std::tie(attr_state->name_, attr_state->stride_,
-                     attr_state->offset_) =
+            std::string target;
+            int stride, offset;
+
+            std::tie(target, stride, offset) =
                     GetNameStrideOffsetForAttribute(attr_name);
 
-            if (primary_attr_init.count(attr_state->name_)) {
-                if (primary_attr_init.at(attr_state->name_) == false) {
+            if (target == "f_rest") {
+                stride = 1;
+            }
+
+            attr_state->name_ = target;
+            attr_state->stride_ = stride;
+            attr_state->offset_ = offset;
+
+            if (primary_attr_init.count(target)) {
```

Comment:
What is the purpose of this change?

Reply:
I explicitly assign target, stride, and offset into the attribute state to decouple the property name mapping from its internal representation. This makes the code clearer and easier to maintain and read.